App/authPopup.js

```diff
@@ -14,11 +14,21 @@ function selectAccount () {
 
     const currentAccounts = myMSALObj.getAllAccounts();
 
-    if (currentAccounts.length === 0) {
+    if (!currentAccounts || currentAccounts.length < 1) {
```

Comment:
`getAllAccounts` should return an empty array if no accounts are found. Was there a reason you had to add a check for truthiness here?

Reply:
Oops, I was trying to align in it with our docs. Will revert this and update the [doc](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/accounts.md)